admin/cedar/views.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.views.generic import ListView, UpdateView
+from django.urls import reverse
+
+from osf.models.cedar_metadata import CedarMetadataTemplate
+from admin.cedar.forms import CedarMetadataTemplateForm
+
+class CedarMetadataTemplateListView(PermissionRequiredMixin, ListView):
+    template_name = 'cedar/list.html'
+    paginate_by = 10
+    paginate_orphans = 1
+    context_object_name = 'cedar_metadata_template'
+    permission_required = 'osf.view_cedarmetadatatemplate'
```

Comment:
Add this permission to [`get_admin_read_permissions`](https://github.com/CenterForOpenScience/osf.io/blob/28a5cd691be1bf7c9fe7876647fe2b7fc68c89a1/osf/migrations/__init__.py#L35)